tests/Unit/Wire/IO/SocketIOTest.php

```diff
@@ -35,6 +36,28 @@ public function connect_ipv6()
         $this->assertEquals(0, $ready);
     }
 
+    public function force_connect_ipv4()
+    {
+        $config = new AMQPConnectionConfig();
+        $config->setSocketProtocolMode(AF_INET);
```

Comment:
I think if we remove this line, tests won't fail. Seems test case is not testing this feature.

Reply:
I'm new to test. Can your help my in wraiting tests?

Reply:
I would also add that:

to be run, your two added functions must have the below headers:
```
    /**
     * @test
     */
```